doc/Prettier.md

```diff
@@ -1,8 +1,11 @@
 # Using markdownlint with Prettier
 
-[`Prettier`](https://prettier.io) is a popular code formatter. For the most part,
-it works seamlessly with `markdownlint`. Special situations are documented
-below.
+[`Prettier`](https://prettier.io) is a popular code formatter.
+For the most part, Prettier works seamlessly with `markdownlint`.
+
+You can extend from the `markdownlint/style/prettier.json` style to disable all Markdownlint rules that overlap with Prettier.
```

Comment:
Please:
- wrap at 80 columns
- lower-case "markdownlint" (breaking CI)
- remove word "from"
- add "rules" to the end: "overlap with Prettier rules"

Reply:
Nit: on the last point, my understanding is Prettier doesn't really have "rules" per se. Unless I'm wrong it's a one-pass AST printing that just has a few configuration settings.